### PR TITLE
[tests/usbdev] Support more targets for usbdev tests

### PR DESCRIPTION
**Pull request by alees24**
In response to earlier comments, this PR drops the hard check of device type now that usbdev tests are more widely usable and prefers to rely upon the build system to constrain the target types.
This should help with SiVal bring up too. Include Cw340 target now that usbdev smoke test passes.

It also strips an incorrect and unnecessary sign-on message from usbdev_pullup_test.
It has been observed that when running the tests on FPGA via Bazel there is already a similar message.

However, the sign-on message still have some utility as a progress indicator in tests still being developed under Verilator, so the streaming tests (which still require some work on the DPI model before they will pass) retain their sign-on messages for now.

### CODE REVIEW

**Review by marnovandermaas — APPROVED** [review]
Looks good to me, let's wait for CI to finish before merging.

**Review by a-will — APPROVED** [review]

**Pull request merged**
